contrib/debian/libfwupd1.symbols

```diff
@@ -0,0 +1,104 @@
+libfwupd.so.1 libfwupd1 #MINVER#
+ fwupd_client_clear_results@Base 0.7.0
```

Comment:
The symbols system (https://www.debian.org/doc/debian-policy/ch-sharedlibs.html#s-sharedlibs-symbols) is meant to live and work specifically at the packaging level.